### PR TITLE
[Sass mode] rework the mode to uniform to CSS / SCSS modes

### PR DESCRIPTION
**Pull request by ficristo**
I was trying to add some tests to the Sass mode, but running locally CodeMirror/test/index.html#sass_* zero test are run.
What am I missing?

### CODE REVIEW

File: mode/sass/test.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+// CodeMirror, copyright (c) by Marijn Haverbeke and others
+// Distributed under an MIT license: http://codemirror.net/LICENSE
+
+(function() {
+  var mode = CodeMirror.getMode({indentUnit: 2}, "sass");
+  function MT(name) { test.mode(name, mode, Array.prototype.slice.call(arguments, 1)); }
+
+  MT("comment", "[comment // this is a comment]")
+})
```
Comment (marijnh) [inline mode/sass/test.js]: You're not calling your namespace function, so the code is never executed.

Comment (ficristo) [inline mode/sass/test.js]: Ops, I missed the obvious... Thank you.

**Comment by ficristo** [comment]
I tryed to uniform the Sass mode to the CSS one, but there are a lot of edge cases, so tests still fail.
Is there a way to reuse the CSS mode even for a braceless language?
If not is there a way to at least reuse its variables somehow, like `propertyKeywords` for example?

_Note to self: check how the indentation works, some tests fail bacause of it._

**Comment by ficristo** [comment]
I disabled the indentation on tests. While probably it could be improved, some test will never pass.
There are still some cases that do not corrispond to css style but for now it's good for me.

**Comment by ficristo** [comment]
Simply rebased and squashed the commits.

**Comment by marijnh** [comment]
Could you make this mode depend on `../css/css.js` and use the enumerated property and value lists from `CodeMirror.mimeModes["text/css"]` instead of copying them here?

**Comment by ficristo** [comment]
Thank you for the guidance. Now depends on CSS mode and indeed I'm happier with it.

**Comment by marijnh** [comment]
Thanks for working on this! Merged as 71af74da

**Comment by ficristo** [comment]
Thank you!
I suppose I have to wait next month for a release, right? I'm not in a hurry, I would like to only have an idea for when will be released.

**Comment by marijnh** [comment]
Yes, I do releases monthly (around the 20th)
